### ISSUE TITLE
Anti ghost unreads/pings

### ISSUE THREAD

**Issue by imadtg** [powercord-community/suggestions#219]
### Description
remove the unread/ping indicators on channels/servers if the unread/ping message is deleted

### More info
you can probably do this by marking the channel as read if the last unread/ping message is deleted but if you could somehow make it less API abusive that'd be nice.

----

 - [x] I checked if this request wasn't already proposed
 - [x] I made sure this request follows Powercord's [plugin guidelines](https://powercord.dev/guidelines)

**Comment by Bowser65** [comment]
Not sure about how feasible this is. I don't exactly remember what the client keeps and what it doesn't during initial load. While it should be fairly easy to do for when the client is running, I think it wouldn't work perfectly and there'd be issues on a cold startup, due to the lack of data, I don't know. I'll mark this as up for grabs since it definitely at least be made less worse.

**Comment by SomeAspy** [comment]
would this be a duplicate of #98?

**Comment by BenSegal855** [comment]
> would this be a duplicate of #98?
> @SomeAspy 

No, this is about clearing the notification when a ping is deleted, #98 is about saving the message when a ping is deleted.
